pydis_site/apps/api/views.py

```diff
@@ -305,7 +305,14 @@ def post(self, request: Request, *, webhook_id: str, webhook_token: str) -> Resp
         )
         headers.pop('Connection', None)
         headers.pop('Content-Length', None)
-        return Response(data=body, headers=headers, status=response_status)
+
+        response_body = {
+            "original_status": response_status,
+            "data": body.decode("utf-8"),
```

Comment:
How does GH know it has to look in this nested structure now instead of the root?

Reply:
It doesn't -- but it doesn't need to, GitHub stores all response details for us to browse, it doesn't do any processing of this response. I'm going to add a sanity log on our side though to mark when we get a 400 from downstream (but without logging sensitive info).